bikeshed/__init__.py

```diff
@@ -1248,6 +1262,7 @@ def preprocess(self):
         processHeadings(self, "all") # again
         addTOCSection(self)
         addSelfLinks(self)
+        processBiblioLinks(self) # again, as biblio links could be added by addIndexSection().
```

Comment:
Hrm. Processing `biblio` links needs to happen before `processAutoLinks`, as they're then transformed such that the latter doesn't attempt to look them up.